components/Tooltip/react/Tooltip.tsx

```diff
@@ -0,0 +1,170 @@
+import clsx from 'clsx'
```

Comment:
https://user-images.githubusercontent.com/14926794/171955516-0f4ed59e-8938-4015-ab91-2020b1d978b8.mov

i'm seeing a little flickering with the react tooltip